jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/MavenProjectEnricherTest.java

```diff
@@ -79,6 +79,45 @@ public void testGeneratedResources() {
         assertNull(selectors.get("version"));
         assertNull(selectors.get("project"));
     }
+    
+    @Test
```

Comment:
This test belongs to a different Suite, please create `ProjectLabelEnricherTest` (remember to add license header, etc.).

It would be nice to have a test for a configured app label and a test for the default app label (by mocking artifactId)

Reply:
Sure. Makes sense.

Reply:
Should I also implment some validation on the APP field?
It doens't make sense with an empty string, and other patterns are likely also not supported by Kubernetes.

Reply:
I don't think it's necessary (if someone overrides this value is because they want to do so purposefully), but go ahead if you want.

You have a helper method in https://github.com/fabric8io/kubernetes-client/blob/25c09cd5d790e55407f999b73502344b2d5f6d49/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/KubernetesResourceUtil.java#L275
which can be called once all labels are computed (i.e. createLabels > createAndValidateLabels)